### DISCUSSION TITLE
Localization of notification content and title

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#212] [General]
The same notifications may be sent to notify users who speak different languages. 
Therefore, we need to provide the notifications in different languages.

However, I could not find any information about the localization of notification content in the documentation.

We can use categories for each language as a workaround but in this case, a separate message should be sent for each language.
It would be nice to have the ability to handle localization in templates. 
For example 
```
if(user.locale == "de") {
  {{ notification.content_de }}
} else if (user.locale == "es") {
 {{ notification.content_de }}
} else {
  {{ notification.content_en }}
}

```

Do you have any suggestions on how to handle localization of notification content?

**Comment by RuslanHryn** [comment]
@unamashana any suggestions?

**Reply by unamashana** [reply]
@RuslanHryn apologies for the delay in responding. There isn't a way to achieve this using the API right now. If you are looking to do this in-app, you can send us content per user, or for email, I suggest using the Ping email provider to get triggers for delivery that you can then handle in your code. 

We hope to have some improvements here in the future, but I am sorry we can't offer a quicker solution at the moment.
